### PR TITLE
Stats report tweaks

### PR DESCRIPTION
**Pull request by mattjmeier**
This could be merged quickly with little consequence. Ready for review

### CODE REVIEW

File: Rmd/stats_report.Rmd [hunk 1]
```diff
@@ -1,31 +1,30 @@
...
+  design: "group"                # single experimental group of interest; entries in this column must match the contrast names.
+  intgroup: ["group"]            # experimental group of interest plus covariates; can be more than one
+  group_facet: "chemical_timepoint"       # If you have many different experimental groups, you may subset the report by specifying a column in the metadata to filter groups, and then setting the group of interest in group_filter
   group_filter:  null        # Which group will this report be done on?
-  display_group_facet: "Chemical_S9" #  
-  display_group_filter:  "AMP +S9" # 
-  sortcol: "Dose"                # Optionally, a column by which to sort the contrasts
+  display_group_facet: "chemical_timepoint" #  
+  display_group_filter:  "estradiol 10d" # 
+  sortcol: "dose"                # Optionally, a column by which to sort the contrasts
```
Comment (mattjmeier) [inline Rmd/stats_report.Rmd:21]: Some day we will have to figure out a way to remember not to commit these changes...
I bet there's a way to automate warnings

**Comment by mattjmeier** [comment]
I also removed some stale parameters, which means I needed to change the test-data repo as well. So main might fail now, but should be fixed once this is merged.

File: Rmd/stats_report.Rmd (hunk 1 above)
Comment (cookkate) [reply Rmd/stats_report.Rmd:22]: But today is not that day!

**Review by cookkate — APPROVED** [review]
Looks good. We should look into improving the summary report since there's a lot of projects lately with lots of facets that could use a summary.

**Pull request merged**